content/_index.md

```diff
@@ -73,40 +69,42 @@ g3Commons:
   c3:
     fig: figs/commons/pdc.png
-    link: /
-  c4:
-    fig: figs/commons/braincommons.png
-    link: https://www.braincommons.org/
+    link: https://proteomic.datacommons.cancer.gov/pdc/
 g3Intro:
   title: Introduction to Gen3
-  detail: Gen3 is an open source software with Apache 2.0 or similar licenses, colocating compute and storage in a data commons. It is agnostic to the data type and the storage location, requiring  minimally, a data model, data, a secure landing page for the portal, and research goals.
+  detail: The Gen3 Data Platform is an open-source (with an Apache License) data platform for developing and operating cloud-based data commons, workspaces, and an ecosystem of interconnected, cloud-based data resources, applications and notebooks. A functioning data ecosystem empowers researchers in the same field to share and combine datasets for reanalysis and reuse as new tools and hypotheses are developed.  The Gen3 Data Platform consists of Gen3 Framework Services, Gen3 Data Commons and Gen3 Workspaces.
   fig: figs/productkv.svg
-  button:
-    caption: Technical Overview
-    link: resources/user
-g3DataAccess:
-  title: Data Access Control on Gen3
-  para1: Gen3 manages data access via internal access control lists stored in a database. It is capable of pulling authorization information from multiple lists, as well as syncing with external sources such as dbGaP. Gen3 supports both users and groups defined in these access control lists - a user that is associated with a group will inherit that group’s permissions.
-  para2: Data is either stored in S3/Google buckets or in our graph database. Only users with "read-storage" and "write-storage" permissions will be able to access stored object data (files) within these buckets, and users with "read" and "write" permissions will be able to access data (clinical data and data that describes a file) in the graph database. These policies prevent public access. Gen3 is also capable of issuing presigned urls for authorized users to directly access objects within buckets, and both the generation of these presigned urls as well as the object downloads themselves are logged.
-  para2Fig: img/figs/data-access-diagram.png
-  para3: Gen3 also has an Attribute Based Access Control (ABAC) engine. It can be used to define data access controls and permissions on a more granular level, and can determine if a user is able to access a specific piece of data. Data in our graph database is modeled as a hierarchy that starts with Programs and Projects. A Program is the root of the tree, and represents an overall group for the data. A Program will have Projects underneath it that consist of any subgroups. Projects can then have many subgroups, and so on. Permissions are generally given on the Program level, but with the ABAC engine, Gen3 can provide more specific authorization that applies to nodes further down the tree.
-  para3Fig: img/figs/data-model-diagram.png
```

Comment:
delete from folder /figs too if not used?: img/figs/data-model-diagram  and img/figs/data-access-diagram.png

Reply:
yeah, we may re-use the images and text elsewhere though. shouldn't hurt anything to keep the PNGs in the folder.